keyrings/codeartifact.py

```diff
@@ -1,18 +1,38 @@
 #!/usr/bin/env python
 
+import profile
 import re
 import logging
+import configparser
 
 import boto3
 import botocore
 
 from keyring import backend
 from keyring import credentials
+from keyring.util import platform_ as platform
```

Comment:
The `keyring.util.platform_` package has been suffixed with an underscore to disambiguate it from the Python 3 built-in `platform` package. Instead of shadowing that built-in package, we should import only the function we need:

```python
from keyring.util.platform_ import config_root
```

Reply:
Fair enough. I typically like to import modules rather than putting functions into the global namespace but its totally a preference. I'll update this to be consistent with the other imports.